infra/cifuzz/actions/action.yml

```diff
@@ -9,9 +9,13 @@ inputs:
     description: 'The total time allotted for fuzzing in seconds.'
     required: true
     default: 360
+  failure-allowed:
+    description: 'Set when action should be run in no fail mode.'
+    default: 'true'
```

Comment:
is there a boolean value possible? e.g. I see an integer on line 11, curious if boolean is supported too, that would make the check on line 56 nicer

Reply:
This is just setting a environment variable in a docker image. I have to parse it on read either way, just as I do with fuzz_seconds. But the user can pass in true Boolean values.